stackslib/src/chainstate/stacks/boot/signers.clar

```diff
@@ -1,33 +1,61 @@
 (define-data-var last-set-cycle uint u0)
-(define-data-var stackerdb-signer-slots (list 4000 { signer: principal, num-slots: uint }) (list))
+(define-data-var stackerdb-signer-slots-0 (list 4000 { signer: principal, num-slots: uint }) (list))
+(define-data-var stackerdb-signer-slots-1 (list 4000 { signer: principal, num-slots: uint }) (list))
 (define-map cycle-set-height uint uint)
-(define-constant MAX_WRITES u340282366920938463463374607431768211455)
+(define-constant MAX_WRITES u4294967295)
 (define-constant CHUNK_SIZE (* u2 u1024 u1024))
+(define-constant ERR_NO_SUCH_PAGE u1)
+(define-constant ERR_CYCLE_NOT_SET u2)
+(define-map cycle-signer-set uint (list 4000 { signer: principal, weight: uint }))
 
+;; Called internally by the Stacks node.
+;; Stores the stackerdb signer slots for a given reward cycle.
+;; Since there is one stackerdb per signer message, the `num-slots` field will always be u1.
 (define-private (stackerdb-set-signer-slots 
```

Comment:
I am confused now....set-signer-slots does seem to set the num-slots...but get-signer-slots seems to return the weight...Def need some clarification on naming here I think D:

Reply:
I think it should be `stackerdb-set-signer-slots` and `set-signers`.  The latter is unrelated to the stackerdb subsystem.